src/binding.cc

```diff
@@ -971,8 +971,9 @@ V(ToStringTag)
 V(Unscopables)
 #undef V
 
-const v8::Value* v8__Symbol__Description(const v8::Symbol& self) {
-  return local_to_ptr(ptr_to_local(&self)->Description());
+const v8::Value* v8__Symbol__Description(v8::Isolate* isolate,
+                                         const v8::Symbol& self) {
+  return local_to_ptr(ptr_to_local(&self)->Description(isolate));
```

Comment:
The old form is deprecated: https://github.com/denoland/v8/blob/89ed41a79629a81e2d89e6342c69593d98ba6b31/include/v8-primitive.h#L578